msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -37,6 +37,491 @@ The following table provides a list of APIs available across the Microsoft Teams
+
+   ```json
+    {
+        "app": {
+            "locale": "en-us",
+            "sessionId": "a57af773-73b1-480c-a525-0c9487a51792",
+            "theme": "dark",
+            "parentMessageId": "",
+            "userClickTime": 1678369341888,
+            "host": {
+                "name": "Teams",
+                "clientType": "web",
+                "sessionId": "",
+                "ringId": "general"
+            }
+        },
+        "page": {
+            "frameContext": "meetingStage",
+            "subPageId": "",
+            "isMultiWindow": false,
+            "sourceOrigin": ""
+        },
+        "user": {
+            "id": "",
+            "licenseType": "Anonymous",
+            "loginHint": "",
+            "userPrincipalName": ""
+        },
+        "chat": {
+            "id": "19:meeting_NTZmNTI3ODgtOWZkOS00NjgzLWJhNTMtMDhlNjE5ZjAwYWYx@thread.v2"
+        },
+        "meeting": {
+            "id": "MCMxOTptZWV0aW5nX05UWm1OVEkzT0RndE9XWmtPUzAwTmpnekxXSmhOVE10TURobE5qRTVaakF3WVdZeEB0aHJlYWQudjIjMA=="
+        }
+    }
+   ```
+
+   # [External user](#tab/external-user)
+
+   The following is a JSON payload response in a scheduled meeting for an external user:
+
+   ```json
+    {
+     "app": {
+      "locale": "en-us",
+      "sessionId": "fe4d7f01-e049-4238-8c5a-3c6bcf4517cd",
+      "theme": "default",
+      "iconPositionVertical": 0,
+      "osLocaleInfo": {
+       "platform": "windows",
+       "regionalFormat": "en-us",
+       "shortDate": "M/d/yyyy",
+       "longDate": "dddd, MMMM d, yyyy",
+       "shortTime": "h:mm tt",
+       "longTime": "h:mm:ss tt"
+      },
+      "parentMessageId": "",
+      "userClickTime": 0,
+      "userFileOpenPreference": "inline",
+      "host": {
+       "name": "Teams",
+       "clientType": "desktop",
+       "sessionId": "d6d8d834-b13c-95bb-96bd-3bed74fdad4c",
+       "ringId": "ring1"
+      },
+      "appLaunchId": "4ce26ee3-a6b6-4dad-9484-3b218d5603be"
+     },
+     "page": {
+      "id": "Test",
+      "frameContext": "sidePanel",
+      "subPageId": "",
+      "isFullScreen": false,
+      "isMultiWindow": true,
+      "sourceOrigin": ""
+     },
+     "user": {
+      "id": "aeee11b1-7838-4b4a-8064-12ce5b01258f",
+      "displayName": "",
+      "licenseType": "Unknown",
+      "loginHint": "v-hrajandira@microsoft.com",
+      "userPrincipalName": "v-hrajandira@microsoft.com",
+      "tenant": {
+       "id": "72f988bf-86f1-41af-91ab-2d7cd011db47",
+       "teamsSku": "enterprise"
+      }
+     },
+     "chat": {
+      "id": "19:meeting_Mzg5ZmQwYzItY2U3NS00MDE4LThkZjQtMzNjNWZiMzM5MzNi@thread.v2"
+     },
+     "meeting": {
+      "id": "MCMxOTptZWV0aW5nX016ZzVabVF3WXpJdFkyVTNOUzAwTURFNExUaGtaalF0TXpOak5XWmlNek01TXpOaUB0aHJlYWQudjIjMA=="
+     },
+     "sharePointSite": {
+      "teamSiteUrl": "",
+      "teamSiteDomain": "microsoft.sharepoint.com",
+      "teamSitePath": "",
+      "teamSiteId": ""
+     }
+    }
+
+   ```
+   ---
+
+1. Call type
```

Comment:
Please specify that this is for an in-tenant user

Reply:
Updated that the JSON responses are for in-tenant users for each meeting type.
![image](https://user-images.githubusercontent.com/93579821/231442005-5815b48c-67a7-46c3-bba2-2e2807e43283.png)